ferrocene/doc/user-manual/src/rustfmt/usage.rst

```diff
@@ -30,13 +36,33 @@ If the ``--check`` option is used, ``rusfmt`` checks if the files are well forma
 Exit codes
 ----------
 
-When running with ``--check``, ``rustfmt`` will exit with status code ``0`` if ``rustfmt`` would not make any formatting changes to the input, and status code ``1`` if ``rustfmt`` would make changes. In other modes, ``rustfmt`` will exit with ``1`` if there was some error during formatting (for example a parsing or internal error) and ``0`` if formatting completed without error (whether or not changes were made).
+When running with ``--check``, ``rustfmt`` will exit with status code ``0`` if
+``rustfmt`` would not make any formatting changes to the input, and status code
+``1`` if ``rustfmt`` would make changes. In other modes, ``rustfmt`` will exit
+with ``1`` if there was some error during formatting (for example a parsing or
+internal error) and ``0`` if formatting completed without error (whether or not
+changes were made).
+
+.. _configuration:
+
+Configuration
+-------------
+
+By default ``rustfmt`` formats code according to the
+`Rust Style Guide <../../../doc/style-guide/index.html>`_. The formatting can
```

Comment:
Why do you have the `../doc` here?

Reply:
Good question, probably that is the reason why the link is broken on the published docs (https://ferroussystems.clickup.com/t/8694zvgt3).

Reply:
```suggestion
`Rust Style Guide <../../style-guide/index.html>`_. The formatting can
```